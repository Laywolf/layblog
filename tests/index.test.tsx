import { render, screen } from '@testing-library/react'
import Home from 'pages/index'

describe('Home', () => {
  it('renders a heading', () => {
    const { container } = render(<Home />)

    const heading = screen.getByRole('heading', {
      name: /ここは彼方花雪の 秘密箱です。/i,
    })

    expect(heading).toBeInTheDocument()

    expect(container).toMatchSnapshot()
  })
})
