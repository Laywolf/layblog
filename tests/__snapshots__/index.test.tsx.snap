// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders a heading 1`] = `
<div>
  <h1
    class="title"
  >
    ここは彼方花雪の
    <br />
    秘密箱です。
  </h1>
  <p
    class="description"
  >
    彼方花雪(かなた はなゆき、카나타 하나유키) 라는 
    <br />
    VRC 유저의 개인 블로그 입니다.
  </p>
  <p
    class="description"
  >
    好きな曲 ↓
  </p>
  <div
    class="video-container"
  >
    <iframe
      allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
      allowfullscreen=""
      frameborder="0"
      height="315"
      src="https://www.youtube.com/embed/2b1IexhKPz4"
      title="YouTube video player"
      width="560"
    />
  </div>
</div>
`;
